src/oaklib/cli.py

```diff
@@ -1093,7 +1107,7 @@ def annotate(
             #         ann.subject_source = line
             #         writer.emit(ann)
         else:
-            text = " ".join(words)
+            text = " ".join(tuple(x for x in words if x not in terms_to_remove))
```

Comment:
there is a strange asymmetry here. if a file is passed, it's left to the implementation to apply filter logic. If it's passed as strings then the logic is in the CLI

Reply:
This was just for prototyping and testing. Fixed it. Everything is consolidated to the implementation (`annotate_text`).